core/cli/src/lib.rs

```diff
@@ -627,7 +626,7 @@ where
 	};
 
 	service::chain_ops::export_blocks::<F, _, _>(
-		config, exit.into_exit(), file, As::sa(from), to.map(As::sa), json
+		config, exit.into_exit(), file, from.into(), to.map(Into::into), json
```

Comment:
Could we file an issue for this to fix this properly?

Reply:
i see no reason why not :)

Reply:
Was more a question if you could do this, but now I have done this :)